showcase/package.json

```diff
@@ -124,9 +124,10 @@
     "@angular/platform-browser-dynamic": "~7.2.2",
     "@angular/platform-server": "~7.2.2",
     "@angular/router": "~7.2.2",
-    "@nationalbankbelgium/stark-core": "file:../dist/packages-dist/stark-core/nationalbankbelgium-stark-core-10.0.0-beta.7-e0b3ee03.tgz",
-    "@nationalbankbelgium/stark-rbac": "file:../dist/packages-dist/stark-rbac/nationalbankbelgium-stark-rbac-10.0.0-beta.7-e0b3ee03.tgz",
-    "@nationalbankbelgium/stark-ui": "file:../dist/packages-dist/stark-ui/nationalbankbelgium-stark-ui-10.0.0-beta.7-e0b3ee03.tgz",
+    "@nationalbankbelgium/code-style": "^1.0.1",
```

Comment:
I'm wondering if `code-style` is a dependency or a dev-dependency... what do you think?

Reply:
I think the current dependencies >< devDependencies are not really correct currently. 
We should have a look on it.
Because I'm pretty sure that if we run `npm install --production`, the application won't work 😕